code/modules/mob/living/carbon/alien/alien_defense.dm

```diff
@@ -1,5 +1,8 @@
 /mob/living/carbon/alien/hitby(atom/movable/AM, skipcatch, hitpush, blocked, datum/thrownthing/throwingdatum)
-	..(AM, hitpush = FALSE)
+	if(!skipcatch && in_throw_mode && !HAS_TRAIT(src, TRAIT_HANDS_BLOCKED) && !restrained())
+		throw_mode_off()
+		AM.attack_alien(src)
```

Comment:
I still feel a little sus about this. How many items can aliens actually pick up or catch in general? Do they *need* to be able to catch anything?

Reply:
Catching facehuggers does actually come up, and more stuff will probably be added in the future, especially when space xenoing
Also if you like throw someone at a xeno... what do you expect

Reply:
At that point it feels like we should just add a trait to things that xenos can pick up. Calling the attack_alien on anything thrown still feels a little off to me, but I'm curious what other folks think

Reply:
Guh, I'm torn. Happy for a trait-based approach since there are so few examples here.

Reply:
moved to a trait solution